src/stories/ChoiceFieldset.stories.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react'
+import {Meta} from '@storybook/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+import {BaseStyles, Box, ThemeProvider} from '..'
+import ChoiceFieldset from '../ChoiceFieldset'
+import Item from '../ChoiceFieldset/ChoiceFieldsetListItem'
+import {ComponentProps} from '../utils/types'
+import {ChoiceFieldsetListProps} from '../ChoiceFieldset/ChoiceFieldsetList'
+
+type Args = ComponentProps<typeof ChoiceFieldset> & ChoiceFieldsetListProps
+
+export default {
+  title: 'Forms/ChoiceFieldset',
+  component: ChoiceFieldset,
+  argTypes: {
+    disabled: {
+      defaultValue: false
+    },
+    required: {
+      defaultValue: false
+    },
+    selectionVariant: {
+      options: ['single', 'multiple'],
+      control: {type: 'radio'},
+      defaultValue: 'single'
+    },
+    validationResult: {
+      options: ['invalidChoice', 'validChoice'],
+      control: {type: 'radio'},
+      defaultValue: 'validChoice'
+    },
+    validationMap: {
+      defaultValue: {
+        invalidChoice: 'error',
+        validChoice: 'success'
+      }
+    }
+  },
+  parameters: {controls: {exclude: ['id', 'validationMap', 'validationResult', 'name', 'onSelect']}},
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+export const RadioGroup = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset {...restArgs}>
+    <ChoiceFieldset.Legend>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
```

Comment:
I can't seem to keyboard tab through these items, is this a known issue?

Reply:
You're not supposed to be able to tab through radio inputs with the same name - you use the arrow keys.